packages/dashboard-core-plugins/src/panels/WidgetPanelTypes.ts

```diff
@@ -24,3 +24,5 @@ export type WidgetPanelTooltipProps = {
   /** Children to render within this tooltip */
   children?: ReactNode;
 };
+
+export type WidgetId = string;
```

Comment:
In case you haven't seen it, we now have a `Brand` type (`packages/utils/src/TypeUtils.ts`). It's useful for ensuring things like Ids don't get assigned to the wrong a field expecting a different id type. Usage is:
`type WidgetId = Brand<'WidgetId'>`

That said, it does require some type assertions in certain cases when creating ids, so there are tradeoffs. Just wanted to call it out in case it is a good fit here.

Reply:
Updated